lib/node_modules/@stdlib/math/base/special/trigamma/test/test.native.js

```diff
@@ -0,0 +1,137 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2024 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var resolve = require( 'path' ).resolve;
+var tape = require( 'tape' );
+var incrspace = require( '@stdlib/array/base/incrspace' );
+var isnan = require( '@stdlib/assert/is-nan' );
+var abs = require( '@stdlib/math/base/special/abs' );
+var NINF = require( '@stdlib/constants/float64/ninf' );
+var PINF = require( '@stdlib/constants/float64/pinf' );
+var EPS = require( '@stdlib/constants/float64/eps' );
+var tryRequire = require( '@stdlib/utils/try-require' );
+
+
+// VARIABLES //
+
+var trigamma = tryRequire( resolve( __dirname, './../lib/native.js' ) );
+var opts = {
+	'skip': ( trigamma instanceof Error )
+};
+
+
+// FIXTURES //
+
+var positive = require( './fixtures/cpp/positive.json' );
+var negative = require( './fixtures/cpp/negative.json' );
+
+
+// TESTS //
+
+tape( 'main export is a function', opts, function test( t ) {
+	t.ok( true, __filename );
+	t.strictEqual( typeof trigamma, 'function', 'main export is a function' );
+	t.end();
+});
+
+tape( 'if provided a `NaN`, the function returns `NaN` ', opts, function test( t ) {
```

Comment:
A bunch of test descriptions here have spaces before the closing `'`. Would you please remove them here (and in the main test file if present there as well?)

Reply:
Yes, I have removed them.